starknet/src/execution_strategies/eth_relayer.cairo

```diff
@@ -11,6 +11,17 @@ mod EthRelayerExecutionStrategy {
     #[storage]
     struct Storage {}
 
+    /// Forwards the execution to the L1<->Starknet bridge to the contract located in the `payload` argument.
```

Comment:
Forwards the proposal data to the L1 execution strategy specified in the `payload` via the Starknet<->L1 bridge.

Reply:
85ee323b4503b8cafc928cd2372ae7fd504fad38